scripts/installer/default.config

```diff
@@ -4,7 +4,7 @@ GLASSFISH_USER = dataverse
 GLASSFISH_DIRECTORY = /usr/local/payara5
 GLASSFISH_ADMIN_USER = admin
 GLASSFISH_ADMIN_PASSWORD = secret
-GLASSFISH_HEAP = 2048
+GLASSFISH_HEAP = 4096
```

Comment:
This might be a dumb question but is another option to remove this line? I thought the installer calculated the heap based on what's available, but I could be mistaken.

Reply:
I don't know about removing the line (since it's an available option) but if you'd like I can replace the longstanding sample value with a comment that says "set this to roughly half of the system RAM, in MB" or some such? I don't like hard-coding a value, either...